app/api_topologies.go

```diff
@@ -25,50 +25,58 @@ var (
 		ID:      "pseudo",
 		Default: "hide",
 		Options: []APITopologyOption{
-			{"show", "Show Unmanaged", nil, false},
-			{"hide", "Hide Unmanaged", render.IsNotPseudo, true},
+			{Value: "show", Label: "Show Unmanaged", filter: nil, filterPseudo: false},
+			{Value: "hide", Label: "Hide Unmanaged", filter: render.IsNotPseudo, filterPseudo: true},
 		},
 	}
 )
 
+// ContainerOpts holds the container topology view options such as "all" and "system"
+var ContainerOpts []APITopologyOption
```

Comment:
I don't see a  reason to use global variables, you can simply append to the map.